src/app/components/user/UserWorkspaces/PaginatedUserWorkspaces.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable relay/unused-fields */
```

Comment:
As per pair-review: if we need to keep it (because of pagination meta fields), let's at least have a better comment about it. Ideally, let's try to not need to use it.

Reply:
Alex and I worked together and decided the best way to avoid this is just combine the paginator file and the component file into one, with the pagination fragment more clearly tied to the only component that relies on it and avoiding the relay/unused-fields issue.